components/director/pkg/graphql/schema.graphql

```diff
@@ -1759,6 +1760,7 @@ type Query {
 	- [query formations](examples/query-formations/query-formations.graphql)
 	"""
 	formations(first: Int = 200, after: PageCursor): FormationPage! @hasScopes(path: "graphql.query.formations")
+	formationsForParticipant(participantId: String!): [Formation!]! @hasScopes(path: "graphql.query.formationsForParticipant")
```

Comment:
We usually refer generically to the entities by "object", for example

formationsForObject(objectId: String....

If that object is not part of any formation it is actually not a "participant". Thinking in that direction, what would be the result here in that case?

Reply:
Reminder not to forget to rename it in the places where the scopes are set.